sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -4,12 +4,16 @@
 
 ### Features Added
 
+- Added support for geo-replication capability. ([44272](https://github.com/Azure/azure-sdk-for-java/pull/44272))
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- Deprecated `getOffset()` that returns a `Long`.  Replaced with `getOffsetString()`.
```

Comment:
nit; is it useful if we specify the types this Api belongs to? 


The `getOffset()` method, which returns a `Long`, has been deprecated in `EventData`, `SystemProperties`, and `Checkpoint`. Replaced with `getOffsetString()`.

Reply:
Thanks! Fixed.